pbft/test/pbft_viewchange_test.cpp

```diff
@@ -30,7 +30,7 @@ namespace bzn
         std::shared_ptr<Mockcrypto_base>
         build_pft_with_mock_crypto()
         {
-            std::shared_ptr<Mockcrypto_base> mockcrypto = std::make_shared<Mockcrypto_base>();
+            std::shared_ptr<Mockcrypto_base> mockcrypto = std::make_shared<NiceMock<Mockcrypto_base>>();
```

Comment:
are you 100% you want to squelch all calls?

Reply:
Nope, changed my mind... Removed the Nice Mock and put in the appropriate expect calls.